package.json

```diff
@@ -9,7 +9,8 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "lint": "eslint 'src/**/*.{js,jsx,ts,tsx}'",
-    "format": "prettier --write 'src/**/*.{js,jsx,ts,tsx}'"
+    "format": "prettier --write 'src/**/*.{js,jsx,ts,tsx}'",
+    "cms-sync": "npx ts-node src/scripts/CMSSync"
```

Comment:
Does this perform the actual sync or just compile the Typescript?

Reply:
This one runs the sync